src/contact.ts

```diff
@@ -449,7 +449,7 @@ export class Contact implements Sayable {
    *
    * @example
    * ```ts
-   * const ret = await contact.remark('lijiarui')
+   * const ret = await contact.alias('lijiarui')
    * if (ret) {
    *   console.log(`change ${contact.name()}'s alias successfully!`)
    * } else {
```

Comment:
Maybe we should unify the ' or ` issue.

I see a lot of ' , while having a lot of ` too.

On my laptop, they could both parsed successfully. with npm run doc and 'npm run lint' is okay.